specs/pvs/corla.pvs

```diff
@@ -347,7 +377,10 @@ END election
 % @relation box inherit storage_container
```

Comment:
This comment somehow got disassociated from the line in question, which seems to be 
`%%% the ballot. Ballots are the means by which voters indicate their`
I think the term "means" helps deal with the fact that some types of ballot defined here, like duplicated_ballot are intermediate forms, which may or may not be a record of voter intent.